proxy/auth/aap.go

```
@@ -0,0 +1,183 @@
+		log.Warnf("Failed to logout: %s", err.Error())
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	RemoveCookie(w)
+	response, err := json.Marshal(RedirectResponse{})
+	if err != nil {
+		log.Warnf("Failed to marshal response: %s", err.Error())
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	w.Write(response)
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)